debezium/src/main/scala/akka/stream/alpakka/debezium/scaladsl/DebeziumSource.scala

```diff
@@ -0,0 +1,142 @@
+/*
+ * Copyright (C) 2016-2020 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.stream.alpakka.debezium.scaladsl
+
+import java.util
+import java.util.concurrent.{ExecutorService, Executors}
+
+import akka.NotUsed
+import akka.actor.{ActorSystem, Props}
+import akka.dispatch.ExecutionContexts
+import akka.stream.alpakka.debezium.impl.{CommitterActor, LazyEmbeddedEngine}
+import akka.stream.alpakka.debezium.{CommittableRecord, DebeziumSettings, SimpleRecord}
+import akka.stream.scaladsl._
+import akka.stream.{OverflowStrategy, QueueOfferResult}
+import io.debezium.embedded.Connect
+import io.debezium.engine.DebeziumEngine
+import org.apache.kafka.connect.source.SourceRecord
+
+import scala.collection.JavaConverters._
+import scala.concurrent.{Await, ExecutionContext, Future, Promise}
+
+object DebeziumSource {
+  def custom(settings: DebeziumSettings)(implicit s: ActorSystem): Source[CommittableRecord, NotUsed] = {
```

Comment:
Ok, so, I created this one using an engine that I tried to create based on the `EmbeddedEngine` class. It's basically `EmbeddedEngine`, but without the automatic polling, so I exposed a method `poll` so we can control it within Akka Streams.
I didn't see why create a `GraphStage` here, so I tried to keep it simple.

Reply:
Yes, I don't see much value in a graph stage either.